src/ReverseDiff.jl

```diff
@@ -17,6 +17,8 @@ using StaticArrays
 
 using MacroTools
 
+using ChainRulesCore
```

Comment:
do we need to depend on this?

Reply:
the implementation here allows RD rules to be defined using ChainRules. It doesn't require RD to depend or have access to `rrule` as long as the user's scope has ChainRules loaded.

Reply:
Right.
You *do* need ChainRulesCore in your dependencies, since you need to lock to a particular set of versions of the API
And you already have it loaded from your downstream, so this line will be a no-op.

Also if you wanted you can fix it to not depend on ChainRulesCore being called ChainRulesCore in the users scope.
Or even loaded there.
IIRC you can do that by interpilating the module object into the expression?

Reply:
yes if we depend on CRC we might as well interpolate it in the macro